infra-as-code/bicep/modules/hubNetworking/parameters/hubNetworking.parameters.min.json

```diff
@@ -58,7 +58,7 @@
       "value": {
         "name": "alz-Vpn-Gateway",
         "gatewayType": "Vpn",
-        "sku": "VpnGw1",
+        "sku": "VpnGw1AZ",
```

Comment:
Not sure about this one since it depends. Also if this changes the parameters.all file should also be changed.

Reply:
will look into this

Reply:
reverted param file